spec/forms/candidate_interface/english_gcse_grade_form_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::EnglishGcseGradeForm, type: :model do
+  describe 'validations' do
+
+    context 'when grade is "other"' do
+      let(:form) { subject }
+
+      before { allow(form).to receive(:grade_is_other?).and_return(true) }
+
+      it { is_expected.to validate_presence_of(:other_grade) }
+    end
+
+    context 'when qualification type is GCSE' do
+      context 'multiple GCSEs disabled' do
+        FeatureFlag.deactivate('multiple_english_gcses')
```

Comment:
I have intentionally left out the activated case here. Since the `EnglishGcseGradeForm` is set to change quite a bit in the next ticket it doesn't seem worth adding tests here just yet. I made a start on these and fell in to difficulties with the validation rules with the `grade` and `grades` contexts, so it will also be easier to do after the refactor.